### PR TITLE
Use PSR7 Request for CGI Checks

### PR DESCRIPTION
**Pull request by AydinHassan**
TODO:

- [x] Compare headers

### CODE REVIEW

**Comment by AydinHassan** [comment]
Sup @mikeymike: This is read for review

File: src/Result/CgiOutBodyFailure.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    /**
+     * @return string
+     */
+    public function getExpectedOutput()
+    {
+        return $this->expectedOutput;
+    }
+
+    /**
+     * @return mixed
```
Comment (AydinHassan) [inline src/Result/CgiOutBodyFailure.php]: Fuck you doc block

Comment (mikeymike) [inline src/Result/CgiOutBodyFailure.php]: Yeah doc block :fu:

Comment (mikeymike) [inline src/Result/CgiOutBodyFailure.php]: on a serious note though, you bothered about fixing this ?

Comment (AydinHassan) [inline src/Result/CgiOutBodyFailure.php]: yeah i'll fix it, was just going to wait for the rest of the feedback before I do it

**Comment by mikeymike** [comment]
Can't see anything wrong with it tbh dude, nice work! :+1:

**Pull request merged**